alf/networks/projection_networks.py

```diff
@@ -182,7 +183,7 @@ def _normal_dist(self, means, stds):
             # kl_divergence(TransformedDistribution(Independent, transforms),
             #               TransformedDistribution(Independent, transforms))
             # ````
-            squashed_dist = td.TransformedDistribution(
+            squashed_dist = dist_utils.SquashSpecNormal(
```

Comment:
SquashSpecNormal should do the transformations in its `__init__` (the lines below should be moved to its construction. The API might be like `SquashSpecNormal(base_normal_dist, spec)` and the `action_means` and `action_magnitudes` will be computed inside it.